qfieldcloud_sdk/sdk.py

```diff
@@ -517,6 +734,8 @@ def package_download(
             local_dir: destination directory where the files will be downloaded
             filter_glob: if specified, download only packaged files which match the glob, otherwise download all
             force_download (bool, optional): Download file even if it already exists locally. Defaults to False.
+        Returns:
+            List[Dict]: A list of dictionaries with information about the downloaded files.
```

Comment:
```suggestion
            list[dict[str, Any]]: A list of dictionaries with information about the downloaded files.
```

The dict should also be updated in the return type of the function.